src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -798,14 +904,212 @@ private void PlayWithCurrentSettingPressed()
+            microbeSpeciesHexSizeAverage.ToString("F2", CultureInfo.CurrentCulture),
+            microbeSpeciesHexSizeStandardDeviation.ToString("F2", CultureInfo.CurrentCulture));
+
+        foreach (var organelle in SimulationParameters.Instance.GetAllOrganelles())
+        {
+            var percentage = worldsList.Average(w => w.MicrobeSpeciesOrganelleStatistics[organelle].Percentage);
+            var average = worldsList.Average(w => w.MicrobeSpeciesOrganelleStatistics[organelle].Average);
+            bbcode += TranslationServer.Translate("MICROBE_ORGANELLE_STATISTICS").FormatSafe(
+                organelle.Name.Replace('\n', ' '),
+                percentage.ToString("P", CultureInfo.CurrentCulture),
+                average.ToString("F2", CultureInfo.CurrentCulture));
+        }
+
+        allWorldsStatisticsLabel.ExtendedBbcode = bbcode;
+    }
+
+    /// <summary>
+    ///   Stores all data auto-evo exploring tool needs to present a world.
+    /// </summary>
+    private class AutoEvoExploringToolWorld
+    {
+        /// <summary>
+        ///   Local copy of auto-evo configuration. Used to avoid modifying the global one
+        /// </summary>
+        public readonly AutoEvoConfiguration AutoEvoConfiguration;
+
+        /// <summary>
+        ///   The game itself
+        /// </summary>
+        public readonly GameProperties GameProperties;
+
+        /// <summary>
+        ///   This list stores copy of species in every generation.
+        /// </summary>
+        public readonly List<Dictionary<uint, Species>> SpeciesHistoryList;
+
+        public readonly List<Dictionary<int, PatchSnapshot>> PatchHistoryList;
+
+        /// <summary>
+        ///   This list stores all auto-evo results.
+        /// </summary>
+        public readonly List<LocalizedStringBuilder> RunResultsList;
+
+        /// <summary>
+        ///   Used to generate world statistics
+        /// </summary>
+        public readonly Dictionary<OrganelleDefinition, (double Percentage, double Average)>
+            MicrobeSpeciesOrganelleStatistics;
+
+        /// <summary>
+        ///   The current generation auto-evo has evolved
+        /// </summary>
+        public int CurrentGeneration;
+
+        /// <summary>
+        ///   Total used auto-evo time
+        /// </summary>
+        public TimeSpan TotalTimeUsed;
+
+        public AutoEvoExploringToolWorld(IAutoEvoConfiguration configuration)
+        {
+            AutoEvoConfiguration = configuration.Clone();
+            GameProperties = GameProperties.StartNewMicrobeGame(new WorldGenerationSettings
+                { AutoEvoConfiguration = AutoEvoConfiguration });
+            SpeciesHistoryList = new List<Dictionary<uint, Species>>();
+            PatchHistoryList = new List<Dictionary<int, PatchSnapshot>>();
+            RunResultsList = new List<LocalizedStringBuilder>();
+            CurrentGeneration = 0;
+            TotalTimeUsed = TimeSpan.Zero;
```

Comment:
These initial values should be specified directly in the fields above instead of the constructor as that's cleaner in my opinion and leaves only the logic that truly needs to be in the constructor in the constructor.